specification/digitaltwins/resource-manager/Microsoft.DigitalTwins/stable/2020-12-01/digitaltwins.json

```diff
@@ -643,6 +954,24 @@
           "type": "string",
           "x-nullable": true,
           "readOnly": true
+        },
+        "privateEndpointConnections": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnection"
```

Comment:
lets not do this. This is a resource array inside another resource. there are two problems - 1. Ensuring the person who is doing get on digital twins has access on the privateEndpointConnection resource. today ARM does the RBAC filtering on the response and removes specific items from the collection to which the caller doesnt have access to. when its a property, ARM cant do that. 2. the size of this array can continue to grow, and so it will have problem with response payload size (it can keep growing) and paginating will be hard.

Reply:
@sjiherzig @andyatwork - please take a look at above

Reply:
Re pagination concerns: The array will contain 10 elements at max.

As for the nesting: the PrivateLink team maintains tools that expect the nested object to be present on main resources in this particular fashion. I understand the rationale behind your suggestion, but that would have to be adopted by the PrivateLink team first, which maintains these tools. Such a change that lies outside of the scope of what we can determine. We're onboarding PrivateLink within the parameters given to us under today's API requirements.

Reply:
For reference, here's another signed-off service (IoTHub): https://github.com/Azure/azure-rest-api-specs/blob/eea7c0141c0b7ad9f66f8ba06560b549c6b3b014/specification/iothub/resource-manager/Microsoft.Devices/stable/2020-08-01/iothub.json#L2297 

The same should be true for any service that has onboarded to PrivateLink per their current spec.